src/EVault/IEVault.sol

```diff
@@ -344,28 +344,43 @@ interface IGovernance {
     /// @notice Retrieves supply and borrow caps in AmountCap format
     function caps() external view returns (uint16 supplyCap, uint16 borrowCap);
 
-    /// @notice Retrieves regular LTV, set for the collateral, which is used to determine the health of the account
-    function borrowingLTV(address collateral) external view returns (uint16);
+    /// @notice Retrieves the borrow LTV of the collateral, which is used to determine if the account is healthy when originating a position
```

Comment:
I think that "originating a position" is the wrong term. The `borrowLTV` will be used when `Riskmanager.checkAccountStatus` is executed.

It will perform `LiquidityUtils.checkLiquidity` that calls `getCollateralValue(...)` with `liquidation = false`.

This means that the `borrowLTV` will be used when the user performs any operations that could lower the HF: create a borrow position, increase the borrow position, lower the collateral value and so on.

"originating a position" to me restricts the use case when the user **creates a new** borrow position.

Recommendation: rewrite the natspec documentation describing all the scenarios when the `borrowLTV` will be used